ProductReleaseAnnouncement/product_rel_announcement.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vi: set ft=python :
+"""
+Auto-generate social media posts for Medik8s release announcements by filling in the latest tags and
+release information.
+
+Usage:
+    product_rel_announcement.py --markdown
+    product_rel_announcement.py --html
+    product_rel_announcement.py --html --markdown
+
+Options:
```

Comment:
Can we add another option for specifying a release of not all the operators but only some of them? It would work really nicely when we release all of them together, but that might not be the case
For example `-- operator snr,mdr` would lead to the announcement text of releasing only SNR and MDR.

Is that make sense?

Reply:
It might make sense yes.
Let's also say that with the new message (which I forgot to report), there is just 1 line per operator, so it is also easy to remove what's not released.

Reply:
I updated the message showing the the usage of the script

https://github.com/medik8s/tools/pull/17#issuecomment-2378655945

Reply:
>there is just 1 line per operator, so it is also easy to remove what's not released.

Indeed, I thought that if we begin automating the message, then extending it to the above use case would make sense.